modules/troubleshoot/pages/debug-bundle/configure/linux.adoc

```diff
@@ -0,0 +1,6 @@
+= Configure Debug Bundle Management in Linux
+:description: Learn how to configure debug bundle retention and storage in Linux environments.
```

Comment:
It's worth mentioning that this configuration only applies to remotely-generated Debug Bundles.

The auto-delete only works for the Redpanda-generated one. If you are inside the broker and run `rpk debug bundle`, the bundle will be generated in the $PWD by default (if possible).